src/e2e/java/teammates/e2e/pageobjects/StudentProfilePage.java

```diff
@@ -159,12 +159,29 @@ public void showPictureEditor() {
         waitForUploadEditModalVisible();
     }
 
-    public void verifyPhotoSize(String height, String width) {
-        assertEquals(height, browser.driver.findElement(By.className("profile-pic")).getCssValue("height"));
-        assertEquals(width, browser.driver.findElement(By.className("profile-pic")).getCssValue("width"));
+    public void closePictureEditor() {
         click(uploadEditModal.findElement(By.className("close")));
     }
 
+    public void verifyPhotoSize(int height, int width) {
+        String browserHeight = browser.driver.findElement(By.className("profile-pic")).getCssValue("height");
+        String browserWidth = browser.driver.findElement(By.className("profile-pic")).getCssValue("width");
+        float imageHeight = Float.parseFloat(browserHeight.replaceFirst("px$", ""));
+        float imageWidth = Float.parseFloat(browserWidth.replaceFirst("px$", ""));
+        assertEquals(height, imageHeight, 1.0);
+        assertEquals(width, imageWidth, 1.0);
+    }
+
+    public void uploadProfilePicAndVerifyDimensions(String imagePath,
+                                                     int height, int width) {
```

Comment:
This should be on the previous line. Looks kind of weird to have line break when it obviously can fit one line.

Reply:
Oh yeah that seems to be remaining from when I still passed the pageobject into it and lint complained. Should be prettier now